tsconfig.json

```diff
@@ -13,20 +13,14 @@
     "noImplicitAny": true,
     "module": "es6",
     "target": "es6",
-    "lib": ["es2019"],
+    "lib": ["es2019", "dom"],
     "moduleResolution": "node",
     "preserveConstEnums": true,
     "declaration": true,
     "downlevelIteration": true,
     "strictNullChecks": true,
     "allowSyntheticDefaultImports": true,
-    "typeRoots": [
-      "./node_modules/@types",
-      "./src/typings"
-    ]
+    "typeRoots": ["./node_modules/@types", "./src/typings"]
   },
-  "include": [
-    "src/**/*",
-    "test/integration/*"
-  ]
+  "include": ["src/**/*", "test/integration/**/*", "test/unit/**/*"]
```

Comment:
This make ts to add definitions of tests files to the build folder, we need another approach